test/test_util/test_backend.py

```diff
@@ -3,7 +3,7 @@
 import unittest
 import unittest.mock
 
-from autosklearn.util.backend import Backend
+from autosklearn.automl_common.common.utils.backend import Backend
```

Comment:
Shouldn't these tests live in `automl_common` now?

Reply:
They should, however the new backend redoes the tests so I'll change them then.